cli2/morphir-elm.ts

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
```

Comment:
We don't need this. For now we should only have a `morphir` and a `morphir make` command.

Reply:
File is still in the PR